frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -594,16 +589,6 @@ export default function SurveyEdit(): JSX.Element {
                                                 type="secondary"
                                                 className="w-max"
                                                 icon={<IconPlus />}
-                                                sideIcon={
-                                                    surveysMultipleQuestionsAvailable ? null : (
-                                                        <IconLock className="ml-1 text-base text-secondary" />
-                                                    )
-                                                }
-                                                disabledReason={
```

Comment:
i removed both entries because the default should be `null` anyway, is that right?